src/modules/Post.ts

```diff
@@ -23,31 +19,33 @@ const initialState: ModelState<Post> = {
 	error: null,
 }
 
-const post = (state = initialState, action: Action<Post>) =>
+export const postReducer = (state = initialState, action: Action<Post>) =>
 	produce(state, draft => {
 		switch (action.type) {
 			case moduleActionTypes.GET_MODEL:
 				startFetching(draft)
 				break
 			case moduleActionTypes.GET_MODEL_SUCCESS:
-				updateData(draft, action.payload)
+				fetchingSuccess(draft, action.payload)
 				break
 			case moduleActionTypes.GET_MODEL_FAIL:
 				endWithError(draft, action.error)
 				break
 		}
 	})
 
-export const reducer = post
-
 // ------------------------------------
-// Selectors
+// Actions
 // ------------------------------------
 
-export const postSelector = state => state.post || initialState
+const moduleName = 'post'
+
+const {moduleActionTypes, moduleActions} = useModuleActions(moduleName)
```

Comment:
I think this extraction is better done on top of this file, as what we did it before. As when new guy comes in and read thru the previous `export const postReducer...` part, he gotta go down here to find out what is actually `moduleActionTypes`.
May be with defining moduleName as well. I find it easier to read when it is defined on top of the file, as other modules, helpers. The rest should contain only reducer and action.